annotation-processor/src/main/kotlin/pl/touk/krush/source/MappingsGenerator.kt

```diff
@@ -64,9 +64,20 @@ class MappingsGenerator : SourceGenerator {
         }
 
         val associationsMappings = entity.getAssociations(MANY_TO_ONE, ONE_TO_ONE)
-                .filter { assoc -> assoc.mapped }.map { "\t${it.name} = this.to${it.target.simpleName}()"}
-
-        val mapping = (propsMappings + embeddedMappings + associationsMappings).joinToString(",\n")
+                .filter { assoc -> assoc.mapped }
+                .map { "\t${it.name} = this.to${it.target.simpleName}()"}
+
+        val mapping: String
+        if(generateRealReferences){
```

Comment:
please reformat if statements as in the rest of the code (space after "if", "else")